topics/timeseries.md

```diff
@@ -170,6 +170,42 @@ This event is emitted when a list is published.
 | `dimension_3` | The resource ID relating to the content being played |
 | `dimension_4` | User's Globally Unique ID, can be joined to [`public.f_rl_users.talis_guid`]({{ site.baseurl }}/topics/users.html). | `myoVK7wfosXXWlw` |
 
+<br/>
+<a name="player-annotate-createannotation-createcomment"></a>
+
+### Events with class `player.annotate.createannotation.createcomment`
+
+| Column | Description | Example |
```

Comment:
Rather than repeating this for several events where the dimensions are the same, consider multiple classes in the `### Events with class ...` heading.

I've literally just opened a PR for TARL LTI events recently added, and used this method to highlight that there are multiple events that share a common set of dimensions. This means that an individual who is doing some analysis ought to become aware of similar events that they might need to take into acocunt to get a fuller picture of the area they are exploring..

Reply:
good idea I've removed the duplication